plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -155,6 +155,16 @@ immutable mutation_tbl = "CREATE TABLE %s (
     FOREIGN KEY(mp_id) REFERENCES mutation_point(id) ON DELETE CASCADE
     )";
 
+immutable mutation4_tbl = "CREATE TABLE %s (
+    id      INTEGER PRIMARY KEY,
+    mp_id   INTEGER NOT NULL,
+    kind    INTEGER NOT NULL,
+    status  INTEGER NOT NULL,
+    time    INTEGER,
+    eq      INTEGER,
```

Comment:
Do we want to ensure eq is never null? What is the default state? unknown == 0?

Reply:
No, we want it to be set to something all the time. We are thinking 
- non-eq (klee found a testcase to kill the mutant)
- eq (all paths traversed and no one violated the assertion)
- running (could be redundant)
- unknown (not tested, this could be the standard)
- timed out (klee timed out)

Reply:
Looking good. Yes running is probably not needed for now. It can be added in the future as an optimization for work distribution, if needed.

Reply:
For the non-eq it would be really really reaaaally useful to give test data to the user. Help the user kill a mutant. Or help to understand what logic is redundant

Reply:
A side question to think about. How can the timeout be automatically derived?

Reply:
Yes, we are currently doing the parsing from KLEE, where we can obtain test data, so we will include that. 

We were thinking of using the option flags for KLEE, where we can set how long KLEE get before halting execution. This could be dynamically done, i.e. user input or similar, but will probably be set to something like 30 seconds for now. If KLEE halts execution, we can know this by looking for a specific file that KLEE outputs.